dla_cnn/spectra_utils.py

```diff
@@ -26,6 +26,8 @@ def get_lam_data(loglam, z_qso, REST_RANGE):
     ix_dla_range = np.logical_and(lam_rest >= REST_RANGE[0], lam_rest <= REST_RANGE[1])
 
     # ix_dla_range may be 1 pixels shorter or longer due to rounding error, we force it to a consistent size here
+    #don't use following code because size_ix_dla_range can not be constant 
+    """
```

Comment:
Instead of commenting it out, add a flag so that the user
can choose to use it or not.